src/Symfony/Component/HttpFoundation/Session/Storage/Proxy/AbstractProxy.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\HttpFoundation\Session\Storage\Proxy;
 
```

Comment:
shouldn't you add a deprecated notice here ?

Reply:
I had one, but removed it because `AbstractProxy` is extended by `SessionHandlerProxy` and this will make the tests that use `SessionHandlerProxy` to not pass. I can't mark these tests as `legacy`. What's a good solution to have a deprecation notice here and tests pass?